dags/common/parsing/xml_extractors.py

```diff
@@ -76,7 +76,7 @@ def __init__(
 
     def extract(self, article: ET.Element):
         value = self.extraction_function(article)
-        if value:
+        if value is not None:
             return value
```

Comment:
This would be enough IMO.
```suggestion
        if value:
            return value
```

Reply:
https://stackoverflow.com/a/7816439

Reply:
@ParthS007 , sadly is not enough 
<img width="407" alt="Screen Shot 2022-10-27 at 14 11 26" src="https://user-images.githubusercontent.com/36658177/199237423-9dfdcc1d-653f-4177-8f48-268ffb6a937d.png">